### ISSUE TITLE
Support for different Graphviz layouts

### ISSUE THREAD

**Issue by digulla** [sphinx-doc/sphinx#5124]
Subject: Please add support to specify the layout per graph like so:

    .. graphviz:: twopi2.gv
      :graphviz_layout: twopi

or even better:

    .. graphviz:: twopi2.gv
      :layout: twopi

The argument should be passed to `dot` via the `-K` parameter.

The goal is to be able to render this demo from the gallery correctly: https://graphviz.gitlab.io/_pages/Gallery/twopi/twopi2.html

### Environment info
- OS: All
- Python version: All
- Sphinx version:  1.4.6
- Extension: sphinx.ext.graphviz

**Comment by tk0miya** [comment]
`graphviz` directive already supports to switch rendering command with `:graphviz_dot:` option:
```
.. graphviz::
   :graphviz_dot: twopi

   digraph { A -> B, C, D, E }
```

Certainly, `:layout:` is a bit more elegant to choose layout engine to me.

**Comment by digulla** [comment]
Thanks, this works.

**Comment by digulla** [comment]
Oops...

**Comment by tk0miya** [comment]
Fixed by #6430.
Thank you for your proposal.
